plugins/woocommerce/src/StoreApi/Formatters/MoneyFormatter.php

```diff
@@ -23,12 +24,24 @@ public function format( $value, array $options = [] ) {
 			]
 		);
 
-		return (string) intval(
-			round(
-				( (float) wc_format_decimal( $value ) ) * ( 10 ** absint( $options['decimals'] ) ),
-				0,
-				absint( $options['rounding_mode'] )
-			)
-		);
+		// Ensure rounding mode is valid.
+		$rounding_modes           = [ PHP_ROUND_HALF_UP, PHP_ROUND_HALF_DOWN ];
```

Comment:
I think this is probably fine, but since it's effectively part of the public API surface, I'll ask anyway: is there any risk to dropping support for `PHP_ROUND_HALF_EVEN` and `PHP_ROUND_HALF_ODD`?

These were previously allowed and supported in the earlier version of the method.

Reply:
Makes total sense! I started by having those too and removed them without thinking it was previously supported after seeing only these scattered around 🫠
I will re-add them 👍